modules/navier_stokes/doc/content/source/variables/ExtrapolateAtJumpFacesVariable.md

```diff
@@ -0,0 +1,16 @@
+# ExtrapolateAtJumpFacesVariable
```

Comment:
Is somebody doesn't know that we extrapolate on the interfaces of piecewise constant blocks, they might think (based on the name) that we are using extrapolated faces based on cell value and gradient.

Reply:
Yea I don't love the current name. Maybe I should name it like `PiecewiseConstantVariable`?

Reply:
I like that much more!

Reply:
renamed